Makefile

```diff
@@ -64,9 +67,11 @@ package: build
 	@mkdir -p pkg
 	@fpm -t deb -p pkg/ \
 	  -d "libc6 >= ${LIBC_VERSION}" -d "libstdc++6 >= ${LIBSTDCXX_VERSION}" \
+  	  --after-install etc/after-install.sh \
 	  ${FPM_ARGS}
 	@fpm -t rpm -p pkg/ \
 	  -d "glibc >= ${LIBC_VERSION}" -d "libstdc++ >= ${LIBSTDCXX_VERSION}" \
+  	  --after-install etc/rpm-after-install.sh \
```

Comment:
Why add a separate file when you can check for the existence of semodule (similar to the other branching in after-install.sh)